portfolio/src/main/client/src/components/controls/lightbox.ts

```diff
@@ -1,4 +1,5 @@
 import {htmlElement} from '@src/util/html';
+import '@res/style/controls/lightbox.scss';
```

Comment:
Does this just add the css to the DOM head tag?

Reply:
Yep.